devsetup-docker/nginx/styleguide-00-per-site.conf

```diff
@@ -0,0 +1,8 @@
+
+## Example styleguide config, uncomment to enable
+# server {
```

Comment:
should be in project-devsetup template or maybe even better in the kickstart as needed without comments - really special to our setup.